### DISCUSSION TITLE
[FR] - Allow more than one Entry Type for Single Sections

### DISCUSSION THREAD

**Discussion by mike-moreau** [craftcms/cms#15630] [Ideas] [answered]
I understand that one Entry Type for a Single Section makes logical sense, but I suppose the new features in Craft 5 have me truly spoiled :-)

I would love to see multiple Entry Type options for a Single Entry. One example use case would be swapping out different layouts and fields for a home page.

Thanks for considering!

<img width="567" alt="Screenshot 2024-08-29 at 3 58 38 PM" src="https://github.com/user-attachments/assets/9446d43a-8b71-4d36-9ae5-cbd992d075f3">

**Comment by brandonkelly** [comment, marked as answer]
That’s just an artificial limitation, so I went ahead and removed it for Craft 5.4. (3bc239100c235ed20147d37d6f3159f6f73a4615)

**Reply by brandonkelly** [reply]
Craft 5.4.0 is out with that change.
